pytools/tag.py

```diff
@@ -162,9 +162,9 @@ class UniqueTag(Tag):
 # }}}
 
 
-TagsType = FrozenSet[Tag]
-TagOrIterableType = Union[Iterable[Tag], Tag, None]
-T_co = TypeVar("T_co", bound="Taggable")
+TagSet = FrozenSet[Tag]
```

Comment:
Is this required? Writing `FrozenSet[Tag]` seems clearer and not that long.

Reply:
Good point. Dropped.